assets/chat/css/generify.scss

```diff
@@ -62,6 +62,57 @@
     }
 }
 
+@keyframes peek {
+    0% {
+        transform: translate(-110%, 0);
+    }
+    
+    50% {
+        transform: translate(-33%, 0);
+    }
+
+    100% {
+        transform: translate(-110%, 0);
+    }
+}
+@keyframes reverse-peek {
+    0% {
+        transform: translate(110%, 0);
+    }
+    
+    50% {
+        transform: translate(33%, 0);
+    }
+
+    100% {
+        transform: translate(110%, 0);
+    }
+}
```

Comment:
generally `:reverse` reverses time, `:mirror` reverses space... maybe it should be called `:rewind`... anyway the same animation plays just backward. the only reason this was special cased for `:slide` was that the animation already worked backward. if you want to keep it it's up to you but i think it's confusing.